spec/abci/abci++_comet_expected_behavior.md

```diff
@@ -155,9 +155,8 @@ Let us now examine the grammar line by line, providing further details.
 >consensus-round     = proposer / non-proposer
 >```
 
-* For every round, if the local process is the proposer of the current round, CometBFT calls `PrepareProposal`, followed by `ProcessProposal`. 
-These two always come together because they reflect the same proposal that the process
-also delivers to itself. 
+* For every round, if the local process is the proposer of the current round, CometBFT calls `PrepareProposal` and then `ProcessProposal`.
+These two always come together because once a proposal is prepared it is broadcast to all validators, including the proposer itself. In the absence of restarts, `ProcessProposal` is invoked with the proposal returned by `PrepareProposal`.
```

Comment:
```suggestion
These two always come together because once a proposal is prepared it is broadcast to all validators, including the proposer itself. In the absence of crashes, `ProcessProposal` is invoked with the proposal returned by `PrepareProposal`.
```